.helm/starter/values.yaml

```diff
@@ -17,3 +17,5 @@ AWX:
     password: Unset
     sslmode: prefer
     type: unmanaged
+
+controller: {}
```

Comment:
Sorry to be nit-picky but could you add an extra line at the end of the file here please?